code/control/GitDispatcher.php

```diff
@@ -209,9 +209,10 @@ protected function createUpdate() {
 	protected function getGitBranches($project) {
 		$branches = [];
 		foreach ($project->DNBranchList() as $branch) {
+			$commit = $this->project->getCommit($branch->SHA());
 			$branches[] = [
 				'key' => $branch->SHA(),
-				'value' => $branch->Name(),
+				'value' => $branch->Name() . ' (' . $this->project->getCommitMessage($commit) . ')',
```

Comment:
sorry, It was a local test for me to triggering fetching more commit messages to test the cache. Will remove.